tests/test_distribute.cpp

```diff
@@ -1,15 +1,17 @@
+#include <pinocchio/fwd.hpp>
+#include "sot/core/robot-utils.hh"
 #include "pinocchio/multibody/model.hpp"
 #include "pinocchio/parsers/urdf.hpp"
 #include "pinocchio/multibody/data.hpp"
 #include "pinocchio/algorithm/center-of-mass.hpp"
 #include "pinocchio/algorithm/frames.hpp"
 
-#include "sot/core/parameter-server.hh"
 #include "sot/talos_balance/distribute-wrench.hh"
 
 #include <boost/test/unit_test.hpp>
 
 #include "test-paths.h"
+#include "sot/core/parameter-server.hh"
```

Comment:
does this require https://github.com/stack-of-tasks/sot-core/pull/156 ?

Reply:
This is necessary to have it working if you accept sot-core#156